tsconfig.json

```diff
@@ -13,6 +13,7 @@
     "esModuleInterop": true,
     "noImplicitAny": true,
     "paths": {
+      "jest-utils/*": [ "../tests/js/utilities/*" ],
```

Comment:
Similar to WooPay: https://github.com/Automattic/woopay/blob/8962c73aa355f52511b5f87cf04caa7a1233406d/tests/js/jest.config.js#L18